charts/xilinx/fpga-xrt-driver-4.7.11/fpga-xrt-driver.yaml

```diff
@@ -7,16 +7,18 @@ spec:
     name: fpga-xrt-driver
     repository:
       name: example
-      url: file:///charts/xilinx
+      url: cm://fpga-xrt-driver/fpga-xrt-driver-chart
     version: 4.7.11
   namespace: fpga-xrt-driver
   nodeSelector:
     feature.node.kubernetes.io/pci-1d0f.present: "true"
     node.kubernetes.io/instance-type: f1.2xlarge
 
+
   set:
     kind: Values
     apiVersion: sro.openshift.io/v1beta1
+    blub: bla
```

Comment:
I assume this is leftover from debugging ;)

Reply:
You are 100% correct.